clifford/invariant_decomposition.py

```diff
@@ -0,0 +1,149 @@
+"""
+.. currentmodule:: clifford.invariant_decomposition
+
+=====================================================
+invariant_decomposition (:mod:`clifford.invariant_decomposition`)
+=====================================================
+
+.. versionadded:: 1.4.0
+
+
+This file implements the invariant decomposition (aka bivector split) of bivectors into
+mutually commuting orthogonal simple bivectors, based on the method of
+M. Roelfs, Spectroscopic and Geometric Algebra Methods for Lattice Gauge Theory, Chapter 6.
```

Comment:
Can you add a bibliography entry to https://github.com/pygae/clifford/blob/master/docs/refs.bib? Then this becomes
```suggestion
:cite:`roelfs2021thesis`, chapter 6.
```

Reply:
done :)